Added files for analysis

hsantila opened the pull request:
This is the folder for the analysis code (analysis_NMRL3.py) , as requested. The other files are the exemplary experimental and simulation files which I used as a reference for writing the analysis.

ohsOllila commented:
I updated a few lines in analysis_NMRL3.py related to the filenames, atom naming, and output: https://github.com/NMRLipids/MATCH/commit/19535f79fb65ff6830a9ca8dededb2a8c5dc5efc

The same commit also contains results from Slipids and CHARMM with 50% cholesterol and without cholesterol. I think that the script puts the quality of different models in the same order as I would do by looking at the figures, i.e., it seems to work reasonably. I have to do other things now, but I will analyze the rest of the simulations asap. 

Since this is most likely useful for, at least, the NMRlipids III project, it would be useful to have a short description of how the fitness is measured here. This description could be also written directly by commenting on the latest NMRLipids III post: http://nmrlipids.blogspot.com/2018/09/nmrlipids-iii-quantitative-measure-for.html

hsantila commented:
I added a short description as a reply the blog with links to the codes. I
hope it is sufficient & helpful :)

ma 8. lokak. 2018 klo 13.29 ohsOllila (notifications@github.com) kirjoitti:

> I updated a few lines in analysis_NMRL3.py related to the filenames, atom
> naming, and output: 19535f7
> <https://github.com/NMRLipids/MATCH/commit/19535f79fb65ff6830a9ca8dededb2a8c5dc5efc>
>
> The same commit also contains results from Slipids and CHARMM with 50%
> cholesterol and without cholesterol. I think that the script puts the
> quality of different models in the same order as I would do by looking at
> the figures, i.e., it seems to work reasonably. I have to do other things
> now, but I will analyze the rest of the simulations asap.
>
> Since this is most likely useful for, at least, the NMRlipids III project,
> it would be useful to have a short description of how the fitness is
> measured here. This description could be also written directly by
> commenting on the latest NMRLipids III post:
> http://nmrlipids.blogspot.com/2018/09/nmrlipids-iii-quantitative-measure-for.html
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NMRLipids/MATCH/pull/43#issuecomment-427799991>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AMFkilrtMQWx45u6_e2iibxa5t2uhaSCks5uizcfgaJpZM4XH6RM>
> .
>
